test

```diff
@@ -221,6 +222,12 @@ function restart_environment()
     ws enable
 )
 
+function destroy_environment()
```

Comment:
can we go for something a lot less impactful, like
```
docker-compose rm --stop console
```

Reply:
that way the console container doesn't need rebuilding or in mutagen's case a full resync of the volume

Reply:
however... acceptance tests also need the dev dependencies, so I'm not sure this is the answer